core/packages/contracts/.prettierrc

```diff
@@ -7,7 +7,6 @@
           "tabWidth": 4,
           "singleQuote": false,
           "bracketSpacing": true,
-          "explicitTypes": "always",
```

Comment:
Why do we need to remove this?

Reply:
It was deprecated in `prettier-plugin-solidity` 1.0.0. See the body of 4b3565d for details.